### PR TITLE
DALA-455 dont show landing if logged in

### PR DESCRIPTION
**Pull request by MarcTM01**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] There is at least one picture for each story, which was created before coding has started
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

**Comment by ppuetsch** [comment]
Bitte noch:
=> Footer-Section on every page! (Align with UX/UI!)
implementieren.

File: dataland-frontend/src/components/generics/ViewFrameworkBase.vue [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <slot></slot>
       </MarginWrapper>
     </TheContent>
+    <DatalandFooter />
```
Comment (ppuetsch) [inline dataland-frontend/src/components/generics/ViewFrameworkBase.vue:20]: Ich finde, wir sollten das globaler lösen statt jeder Seite den Footer zu verpassen. 
Zum Beispiel: 
* Wir packen den Footer in App.vue - mit einem v-if guard dass er nur gerendert wird, wenn die route nicht "/" ist. 
* Wir packen den Footer in App.vue als named routerView. Im Router setzen wir dann für jede Route welcher Footer verwendet werden soll

Was meinst du?

Comment (MarcTM01) [reply dataland-frontend/src/components/generics/ViewFrameworkBase.vue:20]: Den Punkt sehe ich, mir ist nur erstmal keine schöne Lösung eingefallen. Named routerViews kenne ich noch gar nicht. Das schaue ich mir mal an

Comment (MarcTM01) [reply dataland-frontend/src/components/generics/ViewFrameworkBase.vue:20]: Den Punkt sehe ich, mir ist nur erstmal keine schöne Lösung eingefallen. Named routerViews kenne ich noch gar nicht. Das schaue ich mir mal an.

Comment (ppuetsch) [reply dataland-frontend/src/components/generics/ViewFrameworkBase.vue]: ok

**Pull request merged**